reporter/html.js

```diff
@@ -460,6 +460,18 @@ function storeFixture() {
 	}
 }
 
+function appendFilteredTest() {
+	var testId = QUnit.config.testId;
+	if ( testId && testId.length > 0 ) {
+		return "<div id='qunit-filteredTest'>Running filtered test: " + testId.join(", ") +
```

Comment:
I wonder how useful it really is to show the IDs here. They're also in the URL, and rather useless by themselves.

Duplicating the test name won't be useful, since those are already shown below anyway. How about this? "Rerunning selected tests"